quinn-proto/src/endpoint.rs

```diff
@@ -1031,13 +1031,7 @@ impl slog::Value for Timer {
 }
 
 #[derive(Debug, Copy, Clone, Eq, PartialEq, Hash, Ord, PartialOrd)]
-pub struct ConnectionHandle(pub usize);
-
-impl From<ConnectionHandle> for usize {
-    fn from(x: ConnectionHandle) -> usize {
-        x.0
-    }
-}
+pub struct ConnectionHandle(usize);
```

Comment:
I'm making pretty heavy use of this in #198 to skip an unnecessary `HashMap` at the higher level, so I'll probably want to restore it there.